api/src/main/java/eu/europeana/fulltext/api/service/FTService.java

```diff
@@ -556,35 +526,21 @@ public boolean annoPageExistsByTgtId(
         return annoPageRepository.annoPageExistsByTgtId(datasetId, localId, targetId, lang);
     }
 
-    /** Creates an AnnoPage from the AnnotationPreview object, saving it in the database */
-    public TranslationAnnoPage createAndSaveAnnoPage(AnnotationPreview annotationPreview)
-        throws SubtitleConversionException {
-        TranslationAnnoPage annoPage = subtitleService.createAnnoPage(annotationPreview);
-        resourceRepository.saveResource(annoPage.getRes());
-        return annoPageRepository.saveAnnoPage(annoPage);
-    }
-
     /**
      * Retrieves the AnnoPage with the specified dcId, lcId, pgId and lang
      *
      * @return AnnoPage or null if none found
      */
-    public TranslationAnnoPage getAnnoPageByPgId(
+    public AnnoPage getAnnoPageByPgId(
         String datasetId, String localId, String pgId, String lang) {
-        return annoPageRepository.findTranslationByPageIdLang(datasetId, localId, pgId, List.of(), lang);
+        return annoPageRepository.findOriginalByPageIdLang(datasetId, localId, pgId, List.of(), lang);
     }
 
     /**
      * Drops the TranslationAnnoPage and TranslationResource collections. Can only be successfully
```

Comment:
The Javadoc says it drops TranslationAnnoPage and TranslationResource, but that is no longer the case. Also the safeguard to only trigger this from tests seems to be gone. Why is that? 
I'm also wondering if we can make this more safe. I mean, when someone misconfigures the integration test properties and points to our existing test database on chaos for example, I think it will delete everything there.